lib/rules/font-family-name-quotes/index.js

```diff
@@ -69,28 +128,26 @@ const rule = (primary) => {
 		}
 
 		root.walkDecls(/^font(-family)?$/i, (decl) => {
-			const fontFamilies = findFontFamily(decl.value);
+			let fontFamiliess = MakeMutableFontFamilies(findFontFamily(decl.value), decl);
```

Comment:
[suggest]

- `const` is preferred over `let` here
- `fontFamilyNodes` is more readable than `fontFamilies`

```suggestion
			const fontFamilyNodes = MakeMutableFontFamilies(findFontFamily(decl.value), decl);
```

Reply:
thanks